### ISSUE TITLE
View Installed Content Packs

### ISSUE THREAD

**Issue by DS177** [gchq/stroom#3099]
A way to be able to see what Content Packs (and version) you have loaded in Stroom would be a useful enhancement.
That way you can easily check the compatibility matrix at https://github.com/gchq/stroom-content to see if you have the 
most up-to-date pack version for your Stroom version.

**Comment by at055612** [comment]
This is a tricky one.  A content pack is not concept that exists within stroom.  The content packs are in essence no different to an export made from Stroom (and are generally created in that way).  Once imported into stroom they become independent bits of content and are not a single atomic thing.  Each piece of content from a pack can also be changed once imported into stroom.

I think in some places we add date/commit information to the entity description (though that may only be for visualisations).  We could introduce some meta to each content pack and then have Stroom track what has been imported but given that packs can be changed/deleted in part/whole, I am not sure if this would add much value.  I think what you are after would only work if content packs were an immutable atomic thing, but then you would need some way to break open a pack if you wanted to customise it, at which point it would cease to become a pack.

In Summary managing the import/sharing of different versions of content (all with various versioned dependencies) is a very tricky nut to crack.

**Comment by at055612** [comment]
Relates to gchq/stroom-content#11
